tests/model/checkbox.model.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { Checkbox } from '../../src/app/core/models/checkbox.model';
+
+const EXPECTED_CHECKED_STRING = '- [x] todo';
+const EXPECTED_UNCHECKED_STRING = '- [ ] todo';
+
+describe('isValidProfile', () => {
```

Comment:
Be careful of the test name :sweat_smile:

Reply:
oops my bad... updated 😅